server/integration-tests/generate_test_token.hs

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env stack
+-- stack --stack-yaml ../../server/stack.yaml runghc --package QuickCheck --package quickcheck-text -- -Wall
+
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+import Prelude hiding (fail)
+
+import qualified Data.ByteString as SBS
+
+import AccessControl
+import JwtAuth
+import qualified Data.Map.Strict as Map
+import qualified Web.JWT as JWT
+import qualified Data.Text.IO as Text
+
+testAccess :: IcepeakClaim
```

Comment:
Same as above: `testAccess` should go under `testClaims`.
And `main` should be on top.